core/src/rpc.rs

```diff
@@ -112,36 +112,32 @@ impl JsonRpcRequestProcessor {
             Some(config) => config.commitment,
         };
 
-        let slot = match commitment_level {
+        let mut slot = self
+            .block_commitment_cache
+            .read()
+            .unwrap()
+            .slot_with_commitment(commitment_level);
+
+        match commitment_level {
             CommitmentLevel::Recent => {
-                let slot = r_bank_forks.highest_slot();
-                debug!("RPC using working_bank: {:?}", slot);
-                slot
+                if let Some(descendant_slot) = r_bank_forks.highest_descendant(slot) {
+                    debug!("RPC using highest descendant of the last-voted-on slot: {:?}, {:?}", descendant_slot, slot);
```

Comment:
```suggestion
                    debug!("RPC using highest descendant of the last-voted-on slot ({:?}): {:?}", slot, descendant_slot);
```
I'd like to be able to just glance at the end of the log to get the slot being used